vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/text_processing/text_processing_unittest.cc

```diff
@@ -140,27 +143,15 @@ TEST_F(BatAdsTextProcessingPipelineTest, InitValidModelTest) {
   // Act
   ASSERT_TRUE(json_optional.has_value());
   const std::string json = json_optional.value();
-  bool loaded_successfully = text_processing_pipeline.FromJson(json);
+  absl::optional<base::Value> value_optional = base::JSONReader::Read(json);
+  ASSERT_TRUE(value_optional.has_value());
+  bool loaded_successfully =
+      text_processing_pipeline.FromValue(std::move(*value_optional));
 
   // Assert
   EXPECT_TRUE(loaded_successfully);
 }
 
-TEST_F(BatAdsTextProcessingPipelineTest, InvalidModelTest) {
```

Comment:
Why are we removing this case?

Reply:
it's not applicable now, as we get base::Value as input

Reply:
Looks like we need to add (or check that it exists) a case with invalid json in another place.

Reply:
Added more tests to BatAdsTextClassificationResourceTest fixture